json-lint/package.json

```diff
@@ -8,7 +8,7 @@
 		"test": "test"
 	},
 	"scripts": {
-		"test": "jasmine JASMINE_CONFIG_PATH=./test/spec/support/jasmine.json JSONLINT_TEST=true",
+		"test": "../node_modules/.bin/jasmine JASMINE_CONFIG_PATH=./test/spec/support/jasmine.json JSONLINT_TEST=true",
```

Comment:
Why are you adding the relative path to it? If we have all dependencies we will have `jasmine` as a valid command.

Reply:
Because `jasmine` is a dependency to coding-standards and not json-lint. Because of that if you try to run `npm test` in the json-lint directory, npm can't find jasmine.

Reply:
oh i see!